distribution/docs/src/main/resources/_contents/configuring-contents.adoc

```diff
@@ -260,6 +260,159 @@ The configurations for Security STS LDAP and Roles Claims Handler and Security S
 Use the encryption service, described in <<Encryption Service>>, on the command line to set passwords for your LDAP server.
 Then change the LDAP Bind User Password in the configurations to use the encrypted password.
 
+.STS Ldap Login Configuration
+[cols="1,1m,1,2,1,1" options="header"]
+|===
+|Name
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|LDAP URL
+|ldapUrl
+|String
+|LDAP or LDAPS server and port
+|ldaps://${org.codice.ddf.system.hostname}:1636
+|yes
+
+|StartTLS
+|startTls
+|Boolean
+|Determines whether or not to use StartTLS when connecting via the ldap protocol. This setting is ignored if the URL uses ldaps.
+|false
+|yes
+
+|LDAP Bind User DN
+|ldapBindUserDn
+|String
+|DN of the user to bind with LDAP. This user should have the ability to verify passwords and read attributes for any user.
+|cn=admin
+|yes
+
+|LDAP Bind User Password
+|ldapBindUserPass
+|Password
+|Password used to bind user with LDAP.
+|secret
+|yes
+
+|LDAP Username Attribute
+|userNameAttribute
+|String
+|Attribute used to designate the user's name in LDAP. Usually this is uid, cn, or something similar.
+|uid
+|yes
+
+|LDAP Base User DN
+|userBaseDn
+|String
+|Full LDAP path to where users can be found.
+|ou=users\,dc=example\,dc=com
+|yes
+
+|LDAP Base Group DN
+|groupBaseDn
+|String
+|ou=groups\,dc=example\,dc=com
+|Full LDAP path to where groups can be found.
+|yes
+
+|===
+
+===== Configuring STS Claims Handlers
+
+A claim is an additional piece of data about a principal that can be included in a token along with basic token data.
+A claims manager provides hooks for a developer to plug in claims handlers to ensure that the STS includes the specified claims in the issued token.
+
+Claims handlers convert incoming user credentials into a set of attribute claims that will be populated in the SAML assertion.
+For example, the `LDAPClaimsHandler` takes in the user's credentials and retrieves the user's attributes from a backend LDAP server.
+These attributes are then mapped and added to the SAML assertion being created.
+Integrators and developers can add more claims handlers that can handle other types of external services that store user attributes.
+
+.Security STS LDAP and Roles Claims Handler
+[cols="1,1m,1,2,1,1" options="header"]
+|===
+|Name
+|Property
+|Type
+|Description
+|Default Value
+|Required
+
+|LDAP URL
```

Comment:
Ok. Fixed. Got confused because Github linked to the wrong table. Found the bad row & got it straightened out.